plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -500,7 +500,8 @@ class FossId internal constructor(
             }
         }
 
-        val mappedUrlWithoutCredentials = urlProvider.getUrl(urlWithoutCredentials)
+        val mappedUrl = urlProvider.getUrl(urlWithoutCredentials)
```

Comment:
Commit message:
- Typo in "UR*L*"
- I believe we should drop "Previous commit introduced such a bug" as "Previous" is misleading (it's not the parent of this commit) and you anyway mention the commit to be fixed-up.